ui/perfherder/graphs/GraphChangelogTooltip.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import countBy from 'lodash/countBy';
+import moment from 'moment';
+import { Button } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons';
+
+import { alertStatusMap, endpoints } from '../constants';
+import { getJobsUrl, createQueryParams, getApiUrl } from '../../helpers/url';
+import { create } from '../../helpers/http';
+import RepositoryModel from '../../models/repository';
+import { displayNumber, getStatus } from '../helpers';
+
+const GraphChangelogTooltip = ({
+  changelogDataPoint,
+}) => {
+  const data = changelogDataPoint.datum;
+  return (
+    <div className="body">
```

Comment:
You're using an outdated version of this code; the tooltip has been changed to render as an svg. Please rebase on master and implement this the way the GraphTooltip is currently implemented.